packages/client/components/ActionMeetingUpdates.tsx

```diff
@@ -93,6 +98,11 @@ export default createFragmentContainer(ActionMeetingUpdates, {
       id
       endedAt
       showSidebar
+      localPhase {
```

Comment:
this is a big gotcha in relay.
we set up `localPhase` to be local state
local state is never fetched from the server. you can think of `localPhase` as a memoization of the available domain data.
so to guarantee that the local state includes `stages.isComplete`, we need to also request it from the server.

while this works today, it only works because another component is requesting `isComplete`. if that component no longer requests that field, it won't be available here.

so, on L111 you'll need to add `isComplete`. that way it guarantees it'll be available within the local object.

Reply:
ah very interesting! I've just pushed an update including `isComplete`